src/python/gem5/components/boards/riscv_board.py

```diff
@@ -256,7 +256,7 @@ def generate_device_tree(self, outdir: str) -> None:
             root.append(node)
 
         node = FdtNode(f"chosen")
-        bootargs = " ".join(self.get_default_kernel_args())
+        bootargs = self.workload.boot_args
```

Comment:
In #630, workload.boot_args was renamed to workload.command_line, so we should use workload.command_line here as well.

https://github.com/gem5/gem5/pull/630/files#diff-f50c1a48b07134c503b466730301cb8f8aa3b1c1f86fba977f2eecf350cd37b8L88

```suggestion
        bootargs = self.workload.command_line
```